system-int-test/test/cmr/system_int_test/search/granule_search_by_track_test.clj

```diff
@@ -0,0 +1,265 @@
+(ns cmr.system-int-test.search.granule-search-by-track-test
+  "Search CMR granules by track info, i.e. cycle, pass, tile."
+  (:require
+    [clojure.test :refer :all]
+    [cmr.common.util :as util :refer [are3]]
+    [cmr.system-int-test.data2.core :as d]
+    [cmr.system-int-test.data2.granule :as dg]
+    [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+    [cmr.system-int-test.data2.umm-spec-common :as data-umm-cmn]
+    [cmr.system-int-test.utils.index-util :as index]
+    [cmr.system-int-test.utils.ingest-util :as ingest]
+    [cmr.system-int-test.utils.search-util :as search]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1" "provguid2" "PROV2"}))
+
+(defn- ingest-granule-with-track
+  "ingest a generated granule on the given provider, parent collection and track info"
+  [provider-id coll track]
+  (d/ingest provider-id (dg/granule-with-umm-spec-collection
```

Comment:
move `(dg/granule-with-umm-spec-collection` so that the three parameters to `d/ingest` line up.